src/index.js

```diff
@@ -1 +1,49 @@
 import './style.css';
+
+const tableScores = document.querySelector('.table-scores');
+const submitScore = document.querySelector('.submit-game');
+const name = document.querySelector('.name-input');
+const playerScore = document.querySelector('.score-input');
+const refresh = document.querySelector('.refresh-btn');
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/IPWYQeHR5U8y45jrsCzh/scores/';
+
+const renderLeaderboard = (users) => {
+  const user = users.result;
+  let leaderboard = '';
+  user.forEach((gameUser) => {
+    leaderboard += `<li class='scores'>${gameUser.user}: ${gameUser.score}</li>`;
+    tableScores.innerHTML = leaderboard;
+  });
+};
+
+async function getScores() {
+  const response = await fetch(url);
+  const data = await response.json();
+  renderLeaderboard(data);
+}
+
+refresh.addEventListener('click', () => {
+  getScores();
+});
+
+async function scoreForm() {
```

Comment:
- [ ] Make sure to use ```arrow function```, so your project meets the requirements

Reply:
Done ✔